src/main/java/org/jboss/tools/intellij/openshift/utils/ExecHelper.java

```diff
@@ -47,6 +48,12 @@
 public class ExecHelper {
     private static final ScheduledExecutorService SERVICE = Executors.newScheduledThreadPool(Runtime.getRuntime().availableProcessors());
 
+    private static Map<String, String> proxyEnvironmentVariables;
```

Comment:
Don't like to much that ExecHelper has a state like that.

Reply:
Me neither but I can't see a simpler option, a more complex would be to pass down the proxyEnvironmentVariables to the execute methods which implies a lot of refactoring and did not want to take that path.

Do you see another way?